diff --git a/index.d.ts b/index.d.ts
index c169023f9fd6e5c9deefe954c01822b7e17c306a..33b8865af7f111cc3b43e7cc6b865e5cc93e811e 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -133,7 +133,6 @@ export abstract class Message {
 
     // These are `abstract static`, but that isn't allowed. Subclasses of Message will have these methods and properties
     // and not having them on Message makes using this class for its intended purpose quite difficult.
-    static deserializeBinary(bytes: Uint8Array): Message;
     static deserializeBinaryFromReader(message: Message, reader: BinaryReader): Message;
     static serializeBinaryToWriter(message: Message, writer: BinaryWriter): void;
     static toObject(includeInstance: boolean, msg: Message): {};
